From 493f6ee8563565191fa677b3c99a73b98d56d3a7 Mon Sep 17 00:00:00 2001
From: Volker Kempert <volker.kempert@pixmeter.com>
Date: Thu, 19 Mar 2015 13:27:39 +0100
Subject: [PATCH 1/2] Modify default for video device to LCD

---
 include/configs/mx6qsabre_common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/mx6qsabre_common.h b/include/configs/mx6qsabre_common.h
index 09531f5..b9542b0 100644
--- a/include/configs/mx6qsabre_common.h
+++ b/include/configs/mx6qsabre_common.h
@@ -193,7 +193,7 @@
 	"mmcroot=" CONFIG_MMCROOT "\0" \
 	"smp=" CONFIG_SYS_NOSMP "\0"\
 	"bootargs=console=" CONFIG_CONSOLE_DEV ",115200 " CONFIG_SYS_NOSMP " " \
-	"video=mxcfb0:dev=hdmi,1280x720M@60,bpp=32 "            \
+	"video=mxcfb0:dev=lcd,7inch_LCD,if=RGB565 " \
 	"video=mxcfb1:off fbmem=10M\0" \
 	"mmcargs=setenv bootargs ${bootargs} " \
 		"root=${mmcroot} rootwait rw consoleblank=0\0" \
-- 
1.9.1

