From c030efe38a27a2ef2a8e0097fda850a5cb98d32d Mon Sep 17 00:00:00 2001
From: Volker Kempert <volker.kempert@pixmeter.com>
Date: Sat, 11 Apr 2015 20:28:37 +0200
Subject: [PATCH] Set device tree file name

---
 boards.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/boards.cfg b/boards.cfg
index 66bc214..ad62226 100644
--- a/boards.cfg
+++ b/boards.cfg
@@ -287,7 +287,7 @@ mx6solosabreauto_eimnor      arm         armv7       mx6qsabreauto       freesca
 mx6solosabreauto_nand        arm         armv7       mx6qsabreauto       freescale      mx6		mx6qsabreauto:IMX_CONFIG=board/freescale/mx6qsabreauto/mx6solo.cfg,MX6SOLO,DEFAULT_FDT_FILE="imx6dl-sabreauto.dtb",DDR_MB=1024,SYS_BOOT_NAND,SYS_NOSMP="nosmp"
 mx6solosabreautoandroid            arm         armv7       mx6qsabreauto       freescale      mx6		mx6qsabreauto:IMX_CONFIG=board/freescale/mx6qsabreauto/mx6solo.cfg,MX6SOLO,DEFAULT_FDT_FILE="imx6dl-sabreauto.dtb",DDR_MB=1024,SYS_NOSMP="nosmp",ANDROID_SUPPORT
 mx6solosabreautoandroid_nand   arm         armv7       mx6qsabreauto       freescale      mx6		mx6qsabreauto:IMX_CONFIG=board/freescale/mx6qsabreauto/mx6solo.cfg,MX6SOLO,DEFAULT_FDT_FILE="imx6dl-sabreauto.dtb",DDR_MB=1024,SYS_BOOT_NAND,SYS_NOSMP="nosmp",ANDROID_SUPPORT
-mx6solo_RIoTboard               arm         armv7       mx6solo_RIoTboard         freescale      mx6             mx6solo_RIoTboard:IMX_CONFIG=board/freescale/mx6solo_RIoTboard/mx6solo_RIoTboard.cfg,MX6SOLO,DEFAULT_FDT_FILE="imx6solo_RIoTboard.dtb",DDR_MB=1024,SYS_NOSMP="nosmp"
+mx6solo_RIoTboard               arm         armv7       mx6solo_RIoTboard         freescale      mx6             mx6solo_RIoTboard:IMX_CONFIG=board/freescale/mx6solo_RIoTboard/mx6solo_RIoTboard.cfg,MX6SOLO,DEFAULT_FDT_FILE="imx6dl-riotboard.dtb",DDR_MB=1024,SYS_NOSMP="nosmp"
 mx6slevk                     arm         armv7       mx6slevk            freescale      mx6		mx6slevk:IMX_CONFIG=board/freescale/mx6slevk/imximage.cfg,MX6SL,SYS_USE_SPINOR
 mx6slevk_spinor           arm         armv7       mx6slevk            freescale      mx6		mx6slevk:IMX_CONFIG=board/freescale/mx6slevk/imximage.cfg,MX6SL,SYS_BOOT_SPINOR
 mx6slevkandroid          arm         armv7       mx6slevk            freescale      mx6		mx6slevk:IMX_CONFIG=board/freescale/mx6slevk/imximage.cfg,MX6SL,SYS_USE_SPINOR,ANDROID_SUPPORT
-- 
1.9.1

